.github/workflows/docker-build-push-google.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: build and push docker image to google
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: "github actions runner"
+        required: true
```

Comment:
is it not possible to define a default `strg-infra`

Reply:
This would be ok to have it defined. It can confused people why once it's set and once not. 
I prefer to define it on each place. Or do you see probability that strg-infra will be renamed?

Reply:
possible but would not recommend that. it should be set explicitly.